README.md

```diff
@@ -2203,6 +2203,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Tushar Kashyap](https://github.com/tusharkashyap11)
   - [Tushar Pathak](https://github.com/tushar54594)
   - [Twinkle](https://github.com/twinkletrivaan)
+  - [Taher Patrawala](https://github.com/Taherpatrawala)
```

Comment:
please add yourself in alphabetical order also with the second and third ... letter